src/common/datomish/db.cljc

```diff
@@ -691,7 +668,7 @@
                   (apply str "WITH t(searchid, a, v, value_type_tag) AS (VALUES "
                          (apply str (repeater (count chunk))) ;; TODO: join?
                          ") SELECT t.searchid, d.e
-                           FROM t, datoms AS d
+                           FROM t, all_datoms AS d
```

Comment:
Sadly, this drives performance into the ground.  See https://github.com/mozilla/datomish/issues/90.